admin/src/main/resources/io/buoyant/admin/js/src/combined_client_graph.js

```diff
@@ -89,6 +103,14 @@ define([
           ignoredClients[routerName][client] = false;
         },
 
+        expireClient: function(client) {
+          expiredClients[routerName][client] = true;
```

Comment:
ah, that's an interesting use case. what's the TTL on clients? i suppose the longer it is, the more ok it is to allow the color to change. i guess it's really a question of trading code complexity here for user friendliness in the ui.